doc/users/whats_new/errorbar_offsets.rst

```diff
@@ -0,0 +1,10 @@
+Errorbar plots can shift which points have error bars
+----------------------
+
+Previously, `plt.errorbar()` accepted a kwarg `errorevery` such that the
+command `plt.errorbar(x, y, yerr, errorevery=6)` would add error bars to
+datapoints `x[::6], y[::6]`.
+
+`errorbar()` now also accepts a tuple for `errorevery` such that
+`plt.errorbar(x, y, yerr, errorevery=(a, b))` adds error bars to points
+`x[a%b::b], y[a%b::b]`.
```

Comment:
Does markevery do `[start%N::N]` or `[start::N]`?  The docs suggest the latter, in which case errorevery should do the same.